materials/terminology.md

```diff
@@ -6,49 +6,32 @@
 While for some there might be differences, the terms "computing cluster", "High Performance Computer (HPC)" and "supercomputer" are also often used interchangeably.
 :::
 
-**Cluster**
-A cluster is all resources wired together for the purpose of high performance computing, which includes computational devices, networking devices (switches) and storage devices combined.
+The term **HPC system** is a stand-alone resource for computationally intensive workloads. 
+They are typically comprised of a multitude of integrated processing and
+storage elements, designed to handle high volumes of data and/or large numbers of floating-point
+operations ([FLOPS](https://en.wikipedia.org/wiki/FLOPS)) with the highest possible performance. 
 
-**Node**
-You can roughly think that one **node** is a single computer.
+To support these constraints, an HPC resource must exist in a specific, fixed location: networking
+cables can only stretch so far, and electrical and optical signals can travel only so fast.
 
-**Core**
-A node contains one or more central or graphical processing units (**CPUs** or **GPUs**) with many **cores** plus shared memory.
+**CPUs** are a computer’s tool for actually running programs and calculations. 
 
-**Job**
-When you want to execute a program on the supercomputer, it has to be boxed into an abstraction layer called "job".
+A Graphical Processing Units (**GPU**) does certain linear algebra operations extremely efficiently, such as those encountered when processing computer graphics.
```

Comment:
```suggestion
**GPU** (graphical processing unit) does certain linear algebra operations extremely efficiently, such as those encountered when processing computer graphics. Widely used also in deep learning.
```

Reply:
Maybe something more like they speed up model training in deep learning ?

Reply:
Updated the suggestion